src/_scss/pages/agency/index.scss

```diff
@@ -42,7 +42,7 @@
         padding-left: 8px;
         width: 100%;
         padding-right: 0;
```

Comment:
I think changing padding here to 2px 0 0 8px will make the 'what is this' text level with the 'view outlays' text. 
And the space between the 'what is this' and the caret changes when you open the accordion. There is much more space there when the accordion is closed.